### PR TITLE
Phpactor projectile root detection

### PR DESCRIPTION
**Pull request by Sasanidas**
With this little change we can make phpactor be automatically aware of projectile root

### CODE REVIEW

File: phpactor.el [hunk 1]
```diff
@@ -185,9 +185,11 @@ have to ensure a compatible version of phpactor is used."
 
 (defun phpactor-get-working-dir ()
   "Return working directory of Phpactor."
+  (if (projectile-mode)
+      (projectile-project-root (buffer-file-name))
```
Comment (zonuexe) [inline phpactor.el]: This project does not depend on projectile, so do not call projectile functions. And don't activate `projectile-mode` without your permission.

Comment (zonuexe) [reply phpactor.el]: Maybe this code can be rewritten as:

```el
(if (and (bound-and-true-p projectile-mode)
         (fboundp 'projectile-project-root))
    (projectile-project-root default-directory)
  (directory-file-name
   (expand-file-name
    (or (php-project-get-root-dir) default-directory))))
```

**Comment by zonuexe** [comment]
@Sasanidas 
What root can't be detected without projectile? Please explain why you need this patch.

**Comment by zonuexe** [comment]
@Sasanidas 
The replacement by Projectile was applied to `php-project` instead of `phpactor.el`.
Enabling `php-project-use-projectile-to-detect-root applies`, to use projectile for Phpactor.
If you update php-mode and it doesn't work as expected, please let me know.

Thank you.

File: phpactor.el (hunk 1 above)
Comment (Sasanidas) [reply phpactor.el]: Right! that's better, sorry for the module call, and thank you for add it to the php-mode.

**Pull request closed without merging**